doubly_linked_list/doubly_linked_list.py

```diff
@@ -86,7 +86,19 @@ def remove_from_tail(self):
     self.tail.next = None
 
   def move_to_front(self, node):
-    pass
+    if node.prev is not None:
+      node.prev.next = node.next
+    
+    if node.next is not None:
+      node.next.prev = node.prev
+
+    self.head.prev = node
+    node.prev = None
+    node.next = self.head
+    self.head = node
+
+  def delete(self, node):
+    node.delete()
 
```

Comment:
Hey Farhan! I'm glad you were able to solve the issues you were having with move to front and delete!